configuration/analyzer_config.json

```diff
@@ -697,6 +697,18 @@
     },
     "soft_time_limit": 100
   },
+  "Rendertron": {
+    "type": "observable",
+    "observable_supported": [
+      "url"
+    ],
+    "description": "get screenshot of a web page using rendertron (puppeteer)",
+    "python_module": "rendertron.Rendertron",
+    "requires_configuration": true,
+    "additional_config_params": {},
+    "soft_time_limit": 600,
```

Comment:
isn't this too much maybe?

Reply:
Yep, you are right. This was for testing purpose as the proxy I was using was too slow.
I'll change it to 120s.